### ISSUE TITLE
rpcallowcors for ETH-RPC

### ISSUE THREAD

**Issue by Stonygan** [DeFiCh/ain#2479]
## Summary
On DVM-Side, CORS is working with parameter rpcallowcors:

```
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Access-Control-Allow-Credentials: true
Access-Control-Allow-Methods: POST, GET, OPTIONS
Access-Control-Allow-Headers: Content-Type, Authorization
Content-Type: application/json
```
For the EVM-RPC its not working and atm no switch to activate it.

**Comment by prasannavl** [comment]
`rpcallowcors` param should picked up by the ETH RPC server, which currently isn't. Should be easy to add. Marking it as up-for-grab for now.

**Comment by shohamc1** [comment]
Fixed by #2540
